datayoga-py/datayoga/blocks/std/write/block.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import sys
+from typing import Any, Dict, List, Optional, Tuple
+
+from datayoga.block import Block as DyBlock
+from datayoga.block import Result
+from datayoga.context import Context
+
+logger = logging.getLogger("dy")
+
+
+class Block(DyBlock):
+
+    def init(self, context: Optional[Context] = None):
+        logger.debug(f"Initializing {self.get_block_name()}")
+
+    async def run(self, data: List[Dict[str, Any]]) -> Tuple[List[Dict[str, Any]], List[Result]]:
+        for record in data:
+            sys.stdout.write(f"{record}\n")
```

Comment:
do we have an easy way to print a nice table as an alternate format?

Reply:
why? you would probably like to pipe it to a file and have a newline delimited json as you suggested...

Reply:
will consider this as an option (will report another issue)